src/lang/english.txt

```diff
@@ -1124,6 +1124,8 @@ STR_TERRAIN_TYPE_FLAT                                           :Flat
 STR_TERRAIN_TYPE_HILLY                                          :Hilly
 STR_TERRAIN_TYPE_MOUNTAINOUS                                    :Mountainous
 STR_TERRAIN_TYPE_ALPINIST                                       :Alpinist
+STR_TERRAIN_TYPE_CUSTOM                                         :Custom
+STR_TERRAIN_TYPE_CUSTOM_VALUE                                   :Custom ({NUM})
```

Comment:
`Custom height`? `Custom peak height` ? might be a bit long, but just "Custom (Number)" feels unclear as to what it means

Reply:
I tried so many combinations by now .. without also renaming "Terrain Type", it seems impossible to get a good combination. Going with `Custom height` for now. Better than `Custom`, but not really accurate. Can't have them all :D